src/avfoundation.cs

```diff
@@ -254,6 +255,113 @@ interface AVMutableDateRangeMetadataGroup
 		AVMetadataItem[] Items { get; set; }
 	}
 	
+	[TV (11,0), NoWatch, iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVDepthData
+	{
+		[Static]
+		[Export ("depthDataFromDictionaryRepresentation:error:")]
+		[return: NullAllowed]
+		AVDepthData Create (NSDictionary imageSourceAuxDataInfoDictionary, [NullAllowed] out NSError outError);
+
+		[Export ("depthDataByConvertingToDepthDataType:")]
+		AVDepthData Create (uint depthDataType);
```

Comment:
Are there specific values for the data type or just any uint?  Is there or could there be an enum to use here?